.github/workflows/default.yml

```diff
@@ -7,53 +7,110 @@ jobs:
   unit-tests:
     name: unit-tests
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
         with:
           go-version: "1.17"
-
       - name: Run unit tests
         run: make test
   acc-tests:
     runs-on: ubuntu-latest
     name: acc-tests
     env:
-      REVIZOR_URL: ${{ secrets.REVIZOR_URL }}
-      REVIZOR_TOKEN: ${{ secrets.REVIZOR_TOKEN }}
       SCALR_TOKEN: ${{ secrets.SCALR_TOKEN }}
       UPSTREAM_ID: ${{ github.run_number }}
     steps:
-      - uses: actions/checkout@v2
-      - uses: actions/setup-go@v2
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
         with:
           go-version: "1.17"
+      - name: Clone fatmouse repo
+        uses: actions/checkout@v3
+        with:
+          repository: Scalr/fatmouse
+          path: fatmouse
+          token: ${{ secrets.GH_PAT }}
       - name: Set API_BRANCH
         if: ${{ contains(github.event.head_commit.message, '[API_BRANCH]') }}
         run: echo "API_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
       - name: Set DB_BRANCH
         if: ${{ contains(github.event.head_commit.message, '[DB_BRANCH]') }}
         run: echo "DB_BRANCH=${GITHUB_REF#refs/heads/}" >> $GITHUB_ENV
+      - id: auth
+        uses: google-github-actions/auth@v0
+        with:
+          credentials_json: ${{ secrets.GOOGLE_APPLICATION_CREDENTIALS }}
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v0
+      - name: Copy secrets
+        shell: bash
+        run: |
+          echo "::group::Copy secrets"
+          mkdir ~/.scalr-labs
+          gsutil cp gs://drone_bucket/prod/private.d/.secrets.yaml fatmouse/tacobell/.secrets.yaml
+          gsutil cp gs://drone_bucket/prod/private.d/github.json ~/.scalr-labs/github.json
+          echo "::endgroup::"
+      - name: Configure docker
+        shell: bash
+        run: gcloud auth configure-docker eu.gcr.io
+      - name: Pull python builder
+        shell: bash
+        run: |
+          echo "::group::Pull python builder image"
+          docker pull eu.gcr.io/development-156220/fatmouse/python-builder:master
+          docker tag eu.gcr.io/development-156220/fatmouse/python-builder:master fatmouse/python-builder:master
+          echo "::endgroup::"
+      - name: Get current job log URL
+        uses: Tiryoh/gha-jobid-action@v0
+        id: get-job-id
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          job_name: ${{ github.job }}
+      - name: Generate run tag
+        shell: bash
+        run: echo "RUN_TAG=${{ github.workflow }}-${{ github.job }}-${{ github.run_number }}" >> $GITHUB_ENV
       - name: Create container
         id: create
-        uses: Scalr/gh-action-revizor@master
-        with:
-          command: create
+        shell: bash
+        run: |
+          echo "::group::Create test app"
```

Comment:
nit: since we have a separate step for container creation/deletion, there's probably no point in creating a separate group, it will basically end up in two blocks you have to expand to see the logs